mvp_project/README.md

```diff
@@ -1,109 +1,183 @@
-  - **Технически:** В базе данных каждая запись связывается с дневником через внешние ключи.
+6. **Геймификация**
+   - Достижения и награды за активность.
+   - Баллы за посещение новых мест.
 
-### 4. Автоматическое отслеживание маршрутов
-- **Как пользователь**, я хочу, чтобы приложение автоматически записывало мой маршрут с использованием GPS, чтобы избежать ручного ввода.
-  - **Контекст:** Пользователь включает GPS, и приложение фиксирует передвижения в реальном времени.
-  - **Технически:** FastAPI принимает данные о местоположении через WebSocket или периодические POST-запросы.
+7. **Премиум-функции**
+   - Расширенное хранилище.
+   - Уникальные шаблоны оформления.
+   - Генерация видеоисторий.
 
 ---
 
-## Категория 2: Социальные взаимодействия
+## **Технологии**
+- **Frontend**: ---
+- **Backend**: FastAPI  
+- **База данных**: PostgreSQL  
+- **Облачное хранилище**: AWS S3 (или аналог)  
+- **API для карт**: Google Maps API или Mapbox  
 
-### 5. Обмен дневниками
-- **Как пользователь**, я хочу делиться своими дневниками с друзьями через уникальную ссылку, чтобы они могли увидеть мои путешествия.
-  - **Контекст:** Пользователь нажимает "Поделиться", и приложение генерирует уникальную ссылку.
-  - **Технически:** API создаёт временный токен доступа для предоставления защищённого просмотра дневника.
+---
 
-### 6. Лента активности
-- **Как пользователь**, я хочу видеть обновления друзей в своей ленте, чтобы оставаться в курсе их путешествий.
-  - **Контекст:** Пользователь открывает ленту активности и видит новые записи или дневники друзей.
-  - **Технически:** FastAPI возвращает список обновлений с использованием пагинации для оптимизации.
+## **User Stories**
 
----
+### **Категория 0: Авторизация и регистрация**
+1. **Регистрация нового пользователя**  
+   _Как новый пользователь, я хочу зарегистрироваться, чтобы получить доступ ко всем функциям платформы._  
+   - **Контекст**: Заполнение email, имени, пароля; подтверждение через email.  
+   - **Технически**: FastAPI обрабатывает запрос, хэширует пароль, сохраняет данные в PostgreSQL.  
 
-## Категория 3: Интерактивные функции
+2. **Авторизация зарегистрированного пользователя**  
+   _Как пользователь, я хочу войти в свой аккаунт, чтобы работать с дневниками._  
+   - **Контекст**: Ввод email и пароля, получение доступа через JWT-токен.  
+   - **Технически**: FastAPI проверяет данные и выдаёт токен.  
 
-### 7. Визуализация маршрутов
-- **Как пользователь**, я хочу видеть карту с отмеченными местами, чтобы наглядно отслеживать свои путешествия.
-  - **Контекст:** Пользователь открывает дневник, и карта отображает посещённые точки с медиафайлами.
-  - **Технически:** Используется Google Maps API или Mapbox для рендеринга карт.
+3. **Авторизация через социальные сети**  
+   _Как пользователь, я хочу входить через Google/Facebook для удобства._  
+   - **Контекст**: OAuth2 для авторизации, сохранение данных в PostgreSQL.  

```

Comment:
if you want to try - you can incorporate Supabase as your MVP database, it has this logins build it

Reply:
https://supabase.com/